project/components/resource_request/schemas.py

```diff
@@ -32,6 +32,8 @@ class ResourceRequestSchema(BaseSchema):
     username: str
     requested_for: str
     completed_at: datetime = None
```

Comment:
Please also fix here:
```suggestion
    completed_at: datetime | None = None
```

Reply:
Done